SU-HSDO/suhumsci
# ARCH-323 Full Link Postcard

# READY FOR REVIEW

# Summary
-  Added new display mode for postcard paragraphs to allow linking the entire card.

# Needed By (Date)
- August 8th

# Urgency
- High

# Steps to Test
1. checkout this branch
1. `comopser update --prefer-source`
1. `blt drupal:sync --site=archaeology --sync-files`
1. create a basic page with a row containing 2+ postcards
1. configure those postcards to have a read more link and choose the "vertical linked card" display 
1. verify the entire card is linked to the button you supplied
1. verify both cards are the same height (even if there is a gap at the bottom of one)


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
